.gitignore

```diff
@@ -1,3 +1,6 @@
+# Git
+*.orig
+
```

Comment:
Not sure when this can happen. 🤔 


```suggestion
```

Reply:
It's `git mergetool` detritus but global ignore works